packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -76,12 +80,12 @@ const _useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedP
       return [];
     }
     /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
-    return visibleVideoParticipants.current.length > 0
-      ? visibleVideoParticipants.current
-      : visibleAudioParticipants.current.concat(callingParticipants);
-    return visibleVideoParticipants.current.length > 0
-      ? visibleVideoParticipants.current
-      : visibleAudioParticipants.current;
+    return visibleGridParticipants.current.length > 0
+      ? visibleGridParticipants.current
+      : visibleHorizontalGalleryParticipants.current.concat(callingParticipants);
```

Comment:
Same here i think this should be `overflow` participants?

Reply:
Good point. Let me do that in refactor PR